transforms/convert-htmlbars-import/index.js

```diff
@@ -0,0 +1,14 @@
+const { getParser } = require('codemod-cli').jscodeshift;
+const { addImportStatement, writeImportStatements } = require('../utils');
+// const { getOptions } = require('codemod-cli');
```

Comment:
Let's remove this line, we don't really accept options so it isn't terribly useful for us.